README.md

```diff
@@ -12,12 +12,54 @@ to build the docker container image
 docker-compose build
 ```
 
+## Run server
+
+### Automatically
+#### In background
+
+```
+docker-compose up -d
+```
+#### In foregraund
+```
+docker-compose up
+```
+or
+```
+docker-compose run --service-ports ams
+```
+#### Helpers
+to check whether started:
+```
+docker-compose ps
+```
+to view logs (all redirected from `/opt/adobe/ams/logs` to stdout):
+```
+docker-compose logs
+```
+to exec some command (e.g. `bash`) inside running container
+```
+docker exec -it docker-adobe-media-server_ams_1 bash
+```
+and then feel free to exit from it - server will not stop
+
+#### To disable access log (if too verbose)
+
+* Option 1.
+  Uncomment `#command: /start.sh --disable-access-log`` in `docker-compose.yml`
+  before running `docker-compose up -d` or
+  `docker-compose run --service-ports ams`
+* Option 2. 
+  `docker-compose run --service-ports ams /start.sh --disable-access-log`
+
+### Manually (just for development/experiments/tests)
+
 Run and start bash (we will be running the server from here). Note: We have to use --service-ports because by default, the [`docker-compose run` command does not map ports](https://docs.docker.com/compose/reference/run/).
 ```
 docker-compose run --service-ports ams bash
 ```
 
-to run the server:
+to run the server (with logging to stdout):
```

Comment:
the following commands don't log to stdout in my experience -- did you mean to change the lines below?

Reply:
Mistake, fixed